balancer/balancer.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.11
```

Comment:
Is this right?

Reply:
Removed.

The failures are blocking all of the PRs. This will at least solve that.